db/seeds.rb

```diff
@@ -5,6 +5,14 @@
 
 $instruments_array = ["piano", "clarinet", "violin"]
 
+def create_single_admin(n)
+  admin = Admin.create(
+    email: "admin_#{n}@gmail.com",
+    password: "password"
```

Comment:
then this doesn't allow us to easily change our `create_admin_accounts` method to create `n` admins. your call, but it makes more sense to me to keep our method to support the creation of `n` admins